source/services/encryption/test/EncryptionService.test.tsx

```diff
@@ -0,0 +1,194 @@
+import { NativeModules } from "react-native";
+import deepCopyViaJson from "../../../helpers/Objects";
```

Comment:
You are using `toBeTruthy` in some of your tests. Truthy values in jest and javascript is (as an example)

```js
const testArray = [];
const testObj = {}
```
And I expect that we don't want these kind of values, or do we?

Reply:
Where `toBeTruthy` we don't want those either, but generally we only want to check that it's not `null` or `undefined` and `toBeTruthy` seemed like a good choice for that. It is inconceivable that a public key for example is ever an object or array, however it can be a number, but maybe also a string in the future.

Would it be better to check for the expected type? `""` is a string but should still be considered invalid..